cdap-ui/app/directives/datetime-picker/datetime.less

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@import "../../styles/variables.less";
+
+@track-color: #464a56;
+@thumb-color: white;
+
+@thumb-radius: 50%;
+@thumb-height: 15px;
+@thumb-width: 15px;
+@thumb-shadow-size: 1px;
+@thumb-shadow-blur: 3px;
+@thumb-shadow-color: #111;
+@thumb-border-width: 0;
+@thumb-border-color: white;
+
+@track-width: 100%;
+@track-height: 5px;
+@track-shadow-size: 0;
+@track-shadow-blur: 0;
+@track-shadow-color: #222;
+@track-border-width: 0;
+@track-border-color: black;
+
+@track-radius: 5px;
+@contrast: 0;
+
+my-datetime-picker {
+
+  .datetime-range {
+    width: 400px;
```

Comment:
Do we need a width here? We seem to be setting this in `<datatime-range>` directive. Seems any controller/component using this directive  could set a width. It could be better if we do this in one place.